src/lv_objx/lv_chart.c

```diff
@@ -24,11 +24,22 @@
 #define LV_CHART_AXIS_TO_LABEL_DISTANCE 4
 #define LV_CHART_AXIS_MAJOR_TICK_LEN_COE 1 / 15
 #define LV_CHART_AXIS_MINOR_TICK_LEN_COE 2 / 3
+#define LV_CHART_AXIS_PRIMARY_Y 1
+#define LV_CHART_AXIS_SECONDARY_Y 0
+#define LV_CHART_LABEL_ITERATOR_FORWARD 1
+#define LV_CHART_LABEL_ITERATOR_REVERSE 0
 
 /**********************
  *      TYPEDEFS
  **********************/
 
+typedef struct {
+    const char * list_start;
+    const char * current_pos;
+    uint8_t items_left;
+    uint8_t is_reverse_iter;
+} label_iterator_t;
```

Comment:
Please prefix this (i.e. `lv_chart_label_iterator_t`). That way there won't be conflicts with other types in scope.

Reply:
FIxed by 9fbad74